src/status_im2/contexts/shell/jump_to/constants.cljs

```diff
@@ -61,3 +61,8 @@
 (def ^:const open-screen-with-shell-animation 3)
 (def ^:const close-screen-without-animation 4)
 (def ^:const open-screen-without-animation 5)
+
+;; Floating Screen gesture
+(def ^:const gesture-width 20)
+(def ^:const gesture-fling-right-velocity 2000)
+(def ^:const gesture-fling-left-velocity -1000)
```

Comment:
We can tweak these parameters and change behavior of swipe as per requirement.
I tried to make them as much as IOS swipe navigation, but will update them as per QA team's requirement when feature will be tested.
(Most probably in next PR will enable shell navigation for communities screen, so that testing can be starting)